lib/miq_automation_engine/service_models/miq_ae_service_service_template_transformation_plan_task.rb

```diff
@@ -27,5 +27,14 @@ def transformation_destination(source_obj)
         wrap_results(@object.transformation_destination(source_obj.object_class.find(source_obj.id)))
       end
     end
+
+    def conversion_host=(conversion_host)
+      raise ArgumentError, "conversion_host must be nil or a MiqAeServiceConversionHost" unless conversion_host.nil? || conversion_host.kind_of?(MiqAeMethodService::MiqAeServiceConversionHost)
+      ar_method do
+        return wrap_results(@object.conversion_host = nil) if conversion_host.nil?
```

Comment:
You should not need `wrap_results` here and I'm also confused why you have to save `@object` below but not here.  Seems you would want to go through the same steps.

Would something like this work?
```ruby
def conversion_host=(conversion_host)
  raise ArgumentError, "conversion_host must be nil or a MiqAeServiceConversionHost" unless conversion_host.nil? || conversion_host.kind_of?(MiqAeMethodService::MiqAeServiceConversionHost)
  ar_method do
    conversion_host = ConversionHost.find_by(:id => conversion_host.id) if conversion_host
    @object.conversion_host = conversion_host
    @object.save
  end
end
```

Reply:
Neat. I like the override of `conversion_host`.